src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -201,6 +201,12 @@ internal void BuildAndLogConsoleLog(string msg, LogLevel level)
             {
                 _consoleLogSource?.Log(consoleLog);
             }
+
+            if (WorkerProcessUtilities.IsToolingConsoleJsonLogEntry(msg))
```

Comment:
move this up to an `else if` in the chain above? we shouldn't send this log to `_consoleLogSource` anymore...